File: Dasar-dasar Pemrograman/Semester 1/Python/PR - Membalikkan List.py
def balikan(param):
    if(type(param) != list):
        return None
        
    a = []
    for i in range(len(param)):
        length = len(param) - i - 1
        a.append(param[length])

    print(a)

a = ['mangga', 'nanas', 'pepaya', 'apel']

balikan(a)

File: Dasar-dasar Pemrograman/Semester 1/JavaScript/PR - Membalikkan List.js
function balikan(param) {
    a = [];

    for (i = 0; i < param.length ; i++) {
        a.push(param[param.length - i - 1]);
    }

    console.log(a);
}

a = ['mangga', 'nanas', 'apel', 'pepaya'];

balikan(a);
